examples/config-popular-registries.json

```diff
@@ -0,0 +1,97 @@
+{
+  "distSpecVersion": "1.0.1",
+  "storage": {
+    "rootDirectory": "/data"
+  },
+  "http": {
+    "address": "0.0.0.0",
+    "port": "5000"
+  },
+  "log": {
+    "level": "info"
+  },
+  "sync": {
+    "credentialsFile": "/config/credentials.json",
```

Comment:
no need for this one, all registries are public

Reply:
nevermind, they can be used to not hit rate limit.

Reply:
Hence, done.